src/libcore/intrinsics.rs

```diff
@@ -913,6 +913,7 @@ extern "rust-intrinsic" {
     /// }
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[rustc_const_unstable(feature = "const_transmute")]
```

Comment:
so it appears that this attribute is entirely ignored. Its implementation is in https://github.com/rust-lang/rust/blob/9606f6fa64926a84d82e3c62dbdc57f5c10f756d/src/librustc/middle/lib_features.rs#L57 Maybe it doesn't work inside `extern` blocks? or the `is_unstable_const_fn` somehow doesn't work with intrinsics?